simbad/util/pyrvapi_results.py

```diff
@@ -420,17 +404,25 @@ def create_contaminant_results_tab(self, contaminant_results, contaminant_mr_res
 
             pyrvapi.rvapi_add_section(sec, section_title, tab, 0, 0, 1, 1, False)
 
-            table_title = "Contaminant AMORE Rotation Search Results"
+            table_title = "Contaminant Rotation Search Results"
             pyrvapi.rvapi_add_table1(sec + "/" + table, table_title, 2, 0, 1, 1, 100)
             df = pandas.read_csv(contaminant_results)
             self.create_table(df, table)
 
-            section_title = "AMORE Rotation Search Graphs"
-            uid = str(uuid.uuid4())
-            graph_sec = section_title.replace(" ", "_") + uid
-            graph_widget = "graphWidget" + uid
-            pyrvapi.rvapi_add_section(graph_sec, section_title, tab, 0, 0, 1, 1, True)
-            self.create_graphs(df, graph_sec, graph_widget)
+            if 'CC_F' in df:
+                section_title = "AMORE Rotation Search Graphs"
+                uid = str(uuid.uuid4())
+                graph_sec = section_title.replace(" ", "_") + uid
+                graph_widget = "graphWidget" + uid
+                pyrvapi.rvapi_add_section(graph_sec, section_title, tab, 0, 0, 1, 1, True)
+                self.create_amore_graphs(df, graph_sec, graph_widget)
+            elif 'llg' in df:
+                section_title = "PHASER Rotation Search Graphs"
+                uid = str(uuid.uuid4())
+                graph_sec = section_title.replace(" ", "_") + uid
+                graph_widget = "graphWidget" + uid
+                pyrvapi.rvapi_add_section(graph_sec, section_title, tab, 0, 0, 1, 1, True)
+                self.create_phaser_graphs(df, graph_sec, graph_widget)
```

Comment:
You repeat the exact same code on lines 506-519. Maybe refactor this into a function?